tests/system/providers/google/cloud/ml_engine/example_mlengine.py

```diff
@@ -216,28 +222,38 @@
     # [END howto_operator_gcp_mlengine_get_prediction]
 
     # [START howto_operator_gcp_mlengine_delete_version]
-    delete_version = MLEngineDeleteVersionOperator(
-        task_id="delete-version", project_id=PROJECT_ID, model_name=MODEL_NAME, version_name="v1"
+    delete_version_v1 = MLEngineDeleteVersionOperator(
+        task_id="delete-version-v1",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        version_name="v1",
+        trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
keep trigger_rule out of doc example - as unnecessary addition:
# [START..
operator code..
# [END..
operator.trigger_rule = TriggerRule.ALL_DONE